scripts/zones/Apollyon/mobs/Dee_Wapa_the_Desolator.lua

```diff
@@ -2,52 +2,81 @@
 -- Area: Apollyon CS
 --  Mob: Dee Wapa the Desolator
 -----------------------------------
-require("scripts/globals/limbus");
------------------------------------
+local ID = require("scripts/zones/Apollyon/IDs")
```

Comment:
Will Dee Wapa need a mixin for Astral Flow?

Reply:
Yeah I'm not sure why it's not there, the other 2 have them. 🤔